src/puppet-web/event.ts

```diff
@@ -195,9 +195,13 @@ async function onMessage(
 
       case MsgType.SYS:
         if (m.room()) {
-          Firer.checkRoomJoin.call(this  , m)
-          Firer.checkRoomLeave.call(this , m)
-          Firer.checkRoomTopic.call(this , m)
+          const joinResult  = await Firer.checkRoomJoin.call(this  , m)
+          const leaveResult = await Firer.checkRoomLeave.call(this , m)
+          const topicRestul = await Firer.checkRoomTopic.call(this , m)
+
+          if (!joinResult && !leaveResult && !topicRestul) {
+            log.error('PuppetWebEvent', `checkRoomSystem message: <${m.content()}> not found`)
```

Comment:
It is not an `error` but a `warning`.

Reply:
please forgive for my lazy and using one pr....  thanks...

But I think here using a silence error is better than a warning, because this is a system error, for the error regExp.

Reply:
It should not be an `error` but a `warning` at here.